documentation/smg/images/08_HoraceNightly.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looking at the diagram in the rich view, it says Herbert at the top, but this is Horace

Reply:
That is because it is triggered by the Herbert nightly trigger unconditionally, and by its own nightly trigger conditional on whether anything has changed